pages.nl/common/vimdiff.md

```diff
@@ -0,0 +1,37 @@
+# vimdiff
+
+> Open twee of meer bestanden in `vim` en toon de verschillen.
+> Bekijk ook: `vim`, `vimtutor`, `nvim`.
+> Meer informatie: <https://www.vim.org>.
+
+- Open twee bestanden en toon de verschillen:
+
+`vimdiff {{pad/naar/bestand1}} {{pad/naar/bestand2}}`
```

Comment:
Can you update the same in English page too?

Reply:
> What exactly? It is/was a copy of the English page

The English page doesn't have the path placeholder, I would suggest adding the same over there too.